CLR_DEV9/PSE/CLR_PSE_PluginLog.cs

```diff
@@ -174,7 +175,13 @@ public static void Close()
 
         public static void WriteLine(TraceEventType eType, int logSource, string str)
         {
-            if (sources == null) return;
+            if (sources == null)
+                return;
+            if (DEV9Header.config != null)
+                if  ((!DEV9Header.config.EnableLogging.Error && eType == TraceEventType.Error) ||
+                    (!DEV9Header.config.EnableLogging.Verbose && eType == TraceEventType.Verbose) ||
+                    (!DEV9Header.config.EnableLogging.Information && eType == TraceEventType.Information))
+                    return;
```

Comment:
Not really sure how I should handle this, because as you said, `LogInit()` is called before the configuration file have been loaded. 
When I made this, I was thinking only about how to stop these logs, I was not even thinking about build types, maybe because I can't find a way to select Release or Debug type in VS Code 0_o

Reply:
I haven't used VSCode much myself, but I've made build tasks that should allow you to create release builds. (You will need to open the solution folder instead of the project folder is vscode to use the tasks)

You could either move any calls to `ConfigFile.LoadConf()` before any `LogInit()` (by moving `ConfigFile.LoadConf()` from `Open()` to `Init()` and reordering the calls in `Config()`) or by splitting `LogInit()` into two functions with Initialise logging with reasonable defaults before a second function is called to set logging levels as desired.

Ideally I would like to be able to enable Verbose logging on a per source level as just one source can produce a large amount entries, although the need for verbose logs doesn't come up often.